### ISSUE TITLE
VoicingNumberPicker should be put into common code

### ISSUE THREAD

**Issue by zepumph** [phetsims/gravity-force-lab-basics#317]
I came about this because of https://github.com/phetsims/scenery/issues/1283#issuecomment-992800551, but think that the full solution should wait on @jessegreenberg and my work in https://github.com/phetsims/sun/issues/730. I will commit a work around that passes only non-voicing options to super, and then passes voicing options through later.

**Comment by zepumph** [comment]
I think that this issue should be put on hold until https://github.com/phetsims/sun/issues/730 is progressed further, and we have a more common-code centric solution for AccessibleValueHandler. sound good @jessegreenberg?

**Comment by jessegreenberg** [comment]
This issue can be closed, VoicingNumberPicker no longer exists and GFLBMassControl implements Voicing through its parent options (eventually AccessibleValueHandler). I just tested the spinner and it is working well. Closing.
